sql/src/test/java/io/crate/executor/BytesRefUtilsTest.java

```diff
@@ -76,4 +79,28 @@ public void testConvertSetWithNullValues() throws Exception {
 
         assertThat((String[]) rows[0][0], Matchers.arrayContainingInAnyOrder("foo", null));
     }
+
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testConvertObjectValues() throws Exception {
+        DataType[] dataTypes = new DataType[]{ DataTypes.OBJECT };
+        Object[][] rows = new Object[1][1];
+        rows[0][0] = new HashMap<String, Object>(){{
```

Comment:
I know it's just a test - but please avoid anonymous class creations with HashMaps